sbncode/Calibration/TrackCaloSkimmer_module.cc

```diff
@@ -0,0 +1,460 @@
+////////////////////////////////////////////////////////////////////////
+// Class:       TrackCaloSkimmer
+// Plugin Type: analyzer (art v3_06_03)
+// File:        TrackCaloSkimmer_module.cc
+//
+// Generated at Mon May 17 09:46:34 2021 by Gray Putnam using cetskelgen
+// from cetlib version v3_11_01.
+////////////////////////////////////////////////////////////////////////
+
+#include "TrackCaloSkimmer.h"
+
+#include "art/Utilities/make_tool.h"
+
+// Global functions / data for fitting
+const size_t MAX_N_FIT_DATA = 30;
```

Comment:
Make configurable?

Reply:
I don't think I need to -- one shouldn't be fitting more than the last ~5cm of the track and so there's no case where you would need more than 30 points. Plus it's easier to make static with a fixed size.

Reply:
OK, a little worried that the residual range fit is not configurable ... so I'd recommend at the very least making sure to throw an error if this 30 number is not enough? You may already have that in place, so sorry if you do and I missed it.